molecule/default/tests/test_default.py

```diff
@@ -17,32 +17,67 @@ def test_packages(host):
     distribution = host.system_info.distribution
     codename = host.system_info.codename
 
-    # Docker package
-    if (
-        distribution == "debian" and (codename == "bookworm" or codename is None)
-    ) or distribution == "kali":
-        # Debian Bookworm is not yet supported by the official Docker
-        # package repo
-        #
-        # https://docs.docker.com/engine/install/debian/
-        assert host.package("docker.io").is_installed
-    elif distribution == "fedora":
-        # Only Moby is available for Feodra 32 and 33
-        #
-        # https://docs.docker.com/engine/install/fedora/
-        assert host.package("moby-engine").is_installed
+    if distribution in ["debian"]:
+        if codename in ["stretch"]:
+            assert all(
+                [
+                    host.package(pkg)
```

Comment:
I believe that as long as it is a valid package (and maybe even if it isn't) the `Package` class will be populated with other information and be truthy. I tested this by adding the following lines:

```python
assert host.package("irssi"), "no irssi package information found"
assert host.package("irssi").is_installed is False, "irssi is installed"
assert False, f"{host.package('irssi') is True} :: {host.package('irssi').is_installed} :: {all([host.package('irssi')])} :: {all([host.package('irssi').is_installed])}"
```

and the output when running `molecule test`:

```
>               assert False, f"{host.package('irssi') is True} :: {host.package('irssi').is_installed} :: {all([host.package('irssi')])} :: {all([host.package('irssi').is_installed])}"
E               AssertionError: False :: False :: True :: False
E               assert False
```

```suggestion
                    host.package(pkg).is_installed
```

Reply:
Yes, I was rushing to wrap this up and copy-pasted the same foolishness all over the joint.  Thank you for cleaning it up.